rustworkx/__init__.pyi

```diff
@@ -272,13 +272,13 @@ _DijkstraVisitor = TypeVar("_DijkstraVisitor", bound=visit.DijkstraVisitor)
 class PyDAG(Generic[_S, _T], PyDiGraph[_S, _T]): ...
 
 def distance_matrix(
-    graph: PyGraph | PyDiGraph,
+    graph: PyGraph[Any, Any] | PyDiGraph[Any, Any],
```

Comment:
Instead of `Any` should we be using `_T` and `_S` here? I'm not familiar with pyright but I just always figured we'd use the type var for this.

Reply:
Distance matrix calculates the unweighted distance matrix so the node and edge types don’t matter. We can use _S and _T to narrow it down but that would be more restrictive than we are in practice